src/lexer_tok_utils.c

```diff
@@ -32,32 +32,22 @@ void	ft_free_tok(t_tok *token)
 
 /**
  * @brief Add character to buffer string.
- * 
+ *
  * In case the buffer string is not long enough, its size is doubled.
  * @param c 		Character to be added.
  * @param buf 		Temporary buffer to save as token.
- * @return t_err 	ERR_MALLOC, SUCCESS
+ * @return t_err 	SUCCESS
```

Comment:
Function doesn't need to return anything anymore as return is not caught in ft_partition()

Reply:
changed to void